packages/react-vapor/src/components/popover/examples/PopoverConnectedExamples.1.use.md

```diff
@@ -0,0 +1,12 @@
+## Guidelines
+
+### Do Not
+
+-   Do not add a title when doing so would not add value to the interaction (e.g., if a popover is triggered from an icon button without a label, you need to explain the context in the title).
+-   Do not display more than one Popover at a time.
```

Comment:
What I think is missing. Not sure this is exactly the cases, but we should discuss.
- What interactive element should we use
- Not sure I understand the point about the title (probably because the example isn't complete enough)
- ONE Popover can be triggered open by default to act as a little tutorial when a user visits a page for the first time. 
- About having only one at a time: since they appear on click, it would technically be impossible to have two at once (unless you have two open by default)
- Should we explain how this related/is different from the technical documentation?

Reply:
Here is my two cent on these points:

1. Any
2. I'll see what I can come up with
3. You are describing the rich popover component that is used within the Admin Console. This will not necessarily be used elsewhere and Vapor is meant to remain Product agnistic.
4. I see that this is not clear since I believe that the idea is to avoid having 2 popovers opened in a screen (on different elements). It does not necessarily refer to multiple popovers on the same element.  I'll work to clarify this point.
5. Not sure I understand your point here. Sorry :/

Reply:
I think I missed interpreted the popover for the rich popover actually. Probably because you refer to having various information layout including splits, etc.